README.md

```diff
@@ -87,7 +87,13 @@ pip install localstack
 You can also spin up *LocalStack* without any installation requirements, using Docker:
 
 ```
-docker run -it -p 4567-4580:4567-4580 -p 8080:8080 atlassianlabs/localstack
+docker run -it -p 4567-4581:4567-4581 -p 8080:8080 atlassianlabs/localstack
+```
+
+Or with docker-compose:
+
+```
+docker-compose up
 ```
 
 ... or simply use the `make` target which runs the same command:
```

Comment:
This is nit-picking, but: The statement in line 99 refers back to the command in line 90. Maybe we can move the new instructions for docker-compose below the `make docker-run` part?